challenges/primitive-data-types/user.ts

```diff
@@ -24,6 +24,6 @@ const musicianInfo = ({
 };
 
 musicianInfo({
-  name,
+  name: artistName,
```

Comment:
I don't really care one way or another, but sorta confused on this change.  if it was to avoid the shorthand syntax (which, is totally understandable: I went back-and-forth on this as well) then why not also change `age`?

Reply:
if you paste your challenge into ts playground it clashes with `name` from `lib.dom.ts`

Reply:
ahhhhhh ok.  we can patch that out with the tsconfig but sure